applications/asset_tracker_v2/tests/ui_module/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(ui_module_test)
+
+# Generate runner for the test
+test_runner_generate(src/ui_module_test.c)
+
+# Create mocks
+cmock_handle(../../src/modules/modules_common.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/event_manager.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/dk_buttons_and_leds.h )
+
+# Include target sources
+target_sources(app PRIVATE ../../src/modules/ui_module.c)
+target_sources(app PRIVATE src/ui_module_test.c)
+
+target_include_directories(app PRIVATE .)
+target_include_directories(app PRIVATE ../../src/)
+target_include_directories(app PRIVATE ../../src/modules/)
+target_include_directories(app PRIVATE ../../src/events/)
+target_include_directories(app PRIVATE ${ZEPHYR_BASE}/../nrf/subsys/event_manager)
+target_include_directories(app PRIVATE ${ZEPHYR_BASE}/../nrf/modules/cjson/include)
+target_include_directories(app PRIVATE ${ZEPHYR_BASE}/../nrfxlib/nrf_modem/include/)
+
+# Options that cannot be passed through Kconfig fragments
+target_compile_options(app PRIVATE
```

Comment:
You can use `target_compile_definitions(app PRIVATE CONFIG_ASSET_TRACKER_V2_APP_VERSION_MAX_LEN=20)` instead. Not that it matters much but it is better CMake syntax.

Reply:
Thank you 👍